parser-typechecker/src/Unison/Codebase/ShortBranchHash.hs

```diff
@@ -21,6 +21,9 @@ fromHash :: Int -> Branch.Hash -> ShortBranchHash
 fromHash len =
   ShortBranchHash . Text.take len . Hash.base32Hex . Causal.unRawHash
 
+fullFromHash :: Branch.Hash -> ShortBranchHash
+fullFromHash = ShortBranchHash . Hash.base32Hex . Causal.unRawHash
```

Comment:
`fromHash` requires a length; this one doesn't.  Analogous to `Reference.toShortHash`, but `Branch.Hash` doesn't have its own module.